api/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/api/mapper/ResultatMapper.kt

```diff
@@ -25,12 +25,12 @@ abstract class ResultatMapper<T>(val resultat: Resultat) {
             resultat.NOTIFIKASJON,
             resultat.PERSISTER_IM,
             resultat.TILGANGSKONTROLL
-        ).filterNotNull()
+        )
     }
 
     fun getConstraintViolations(): List<ConstraintViolation> {
         return findAll()
-            .filter { it.error != null && !it.error!!.melding.isBlank() }
+            .filter { it.error != null && it.error!!.melding.isNotBlank() }
```

Comment:
Forslag til endring

`.filter { it.error != null && !it.error?.melding.isNullOrBlank() }`